Formula/lammps.rb

```diff
@@ -35,41 +36,43 @@ class Lammps < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "f3877f35fa8ec630ff60c6cab01ebcc9ff8c9a7da3184618801b9316cd49c99a"
   end
 
+  depends_on "cmake" => :build
   depends_on "pkg-config" => :build
   depends_on "fftw"
   depends_on "gcc" # for gfortran
-  depends_on "jpeg"
+  depends_on "jpeg-turbo"
```

Comment:
Considering migration of `jpeg` to `jpeg-turbo` as mentioned in https://github.com/Homebrew/homebrew-core/pull/103436